Detectors/TRD/reconstruction/include/TRDReconstruction/TrackletsParser.h

```diff
@@ -78,11 +78,11 @@ class TrackletsParser
   void incParsingError(int error)
   {
     if (mOptions[TRDGenerateStats]) {
-      mEventRecords->incParsingError(error, mFEEID.supermodule, mFEEID.side, mStack * constants::NLAYER + mLayer);
+      mEventRecords->incParsingError(error, mFEEID.supermodule, mRobSide, mStack * constants::NLAYER + mLayer);
```

Comment:
`mRobSide` is the same as `mHalfChamberSide` in DigitsParser.h right? Maybe it makes sense to give them the same name?

Reply:
Yes, it goes to what was already in the trackletparser class, its more sane in the rawreader. will change

Reply:
changed.